### PR TITLE
Feature/ganesh

### PR DESCRIPTION
**Pull request by SekarGaneshAppDev**

### CODE REVIEW

File: src/views/vehicleServiceDetails/vehicleServicepage.jsx [hunk 1]
```diff
@@ -91,20 +94,70 @@ class vehicleServicepage extends Component {
...
+      servicePlanID: parseInt(this.state.servicePlanID),
+      requestedServiceDate: this.state.requestedServiceDate,
+      actualServiceDate: this.state.actualServiceDate,
+      serviceOutDate: this.state.actualServiceDate,
+      status: update_type === 'update_status' ? 'Completed' : this.state.status,
+      isTeamsandConditionsAccepted: this.state.isTeamsandConditionsAccepted,
+      remainderMinutes: parseInt(this.state.remainderMinutes),
+      serviceAmount: this.state.serviceAmount,
+      serviceName: this.state.serviceName,
+      servicePriceChartId: parseFloat(this.state.servicePriceChartId)
```
Comment (varunsapp) [inline src/views/vehicleServiceDetails/vehicleServicepage.jsx]: destructure state variables

File: src/views/vehicleServiceDetails/vehicleServicepage.jsx [hunk 2]
```diff
@@ -91,20 +94,70 @@ class vehicleServicepage extends Component {
...
+      status = '',
+      service_price_chartid = '',
+      service_name = '',
+      service_amount = '',
+      remainder_minutes = '',
+      is_teamsand_conditions_accepted = ''
+    } = e.currentTarget.dataset;
     this.setState(prevState => ({
-      editSheduleModal: !prevState.editSheduleModal
+      editSheduleModal: !prevState.editSheduleModal,
```
Comment (varunsapp) [inline src/views/vehicleServiceDetails/vehicleServicepage.jsx]: pls check spellings - schedule

File: src/views/vehicleServiceDetails/vehicleServicepage.jsx [hunk 3]
```diff
@@ -480,26 +582,40 @@ class vehicleServicepage extends Component {
...
                 />
                 <center>
                   <Button color="success">Update</Button>
                 </center>
               </AvForm>
-              <center>
-                <hr />
-                <Button color="success">Complete Schedule </Button>
-              </center>
+              {this.state.status !== 'On Due' ? (
```
Comment (varunsapp) [inline src/views/vehicleServiceDetails/vehicleServicepage.jsx]: destructure state on render method

status should be single word seperated by _ and uppercase.
add status label as we wanna display.

If client requests to change the words On Due to something else. changes will be in database level

status should be : ON_DUE

File: src/views/vehicleServiceDetails/vehicleServicepage.jsx (hunk 2 above)
Comment (SekarGaneshAppDev) [reply src/views/vehicleServiceDetails/vehicleServicepage.jsx]: Fixed

File: src/views/vehicleServiceDetails/vehicleServicepage.jsx (hunk 1 above)
Comment (SekarGaneshAppDev) [reply src/views/vehicleServiceDetails/vehicleServicepage.jsx]: Fixed

**Review by varunsapp — APPROVED** [review]

**Pull request merged**